[김주형] Mission 2 : 모델링 - 1차 수정

BnDC opened the pull request:
- project initial setting

>wecode PR message template 및 체크 리스트 입니다. 
아래 사항들을 전부 작성/체크 하시고 PR 해주세요!

## 수정 사항 간략한 한줄 요약
- 불필요한 주석 삭제
- email field 수정
- password field 수정
- created_at, updated at field 생성
- alignment convention에 맞게 수정
		 
## 수정 사항들 자세한 내용
(여기에 수정 사항에 대한 자세한 내용을 작성해 주세요.)
- email field 수정 : 중복된 email이 발생 하지 않도록, unique 속성 부여
- password field 수정: password 자체가 아니라 hash값이 저장됨으로 max_length 200으로 수정
- 이력 관리를 위해 created_at, updated at field 생성
- alignment: 띄어쓰기로 다른 줄과 line 맞추기 , class 와 다른 코드 사이 한 줄 띄기



## 기타 질문 및 특이 사항
(궁금한 사항들, 하면서 느낀 점들 공유해주실 것이 있으면 자유롭게 작성해 주세요.)
- 이메일 validation을 함수화할 수는 없나요? (예시) 

## 체크 리스트 (아래 사항들이 전부 체크되어야만 merge가 됩니다!)

- [X] 필요한 test들을 완료하였고 기능이 제대로 실행되는지 확인 하였습니다.
- [X] Wecode의 코드 스타일 가이드에 맞추어 코드를 작성 하였습니다.
- [X] 제가 의도한 파일들과 수정 사항들만 커밋이 된 것을 확인 하였습니다.
- [X] 본 수정 사항들을 팀원들과 사전에 상의하였고 팀원들 모두 해당 PR에 대하여 알고 있습니다.

BnDC committed:
[ADD] User Model

lordmyshepherd commented on students/juhyeong/users/models.py:
```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+# Create your models here.
```
불필요한 주석은 지워주세요~
```suggestion

```

BnDC replied:
수정하였습니다~

lordmyshepherd commented on students/juhyeong/users/models.py:
```diff
@@ -0,0 +1,12 @@
+
+# Create your models here.
+class User(models.Model):
+    name=models.CharField(max_length=50)
+    email=models.CharField(max_length=200)     
+    password=models.CharField(max_length=50)    
+    phone_number=models.CharField(max_length=11)    
+    profile=models.CharField(max_length=255)
```
Alignment 맞춰주세요~
```suggestion
    name  = models.CharField(max_length=50)
    email = models.CharField(max_length=200)
    ..
```

BnDC replied:
수정하였습니다

lordmyshepherd commented on students/juhyeong/users/models.py:
```diff
@@ -0,0 +1,12 @@
+# Create your models here.
+class User(models.Model):
+    name=models.CharField(max_length=50)
+    email=models.CharField(max_length=200)     
```
email field에 `unique` 속성을 추가하셔서 유저가 중복 저장되는 문제가 발생하지 않도록 수정해주세요!  
unique 속성을 어떻게 적용하는지는 공식 문서에서 찾아보세요~

lordmyshepherd commented on students/juhyeong/users/models.py:
```diff
@@ -0,0 +1,12 @@
+class User(models.Model):
+    name=models.CharField(max_length=50)
+    email=models.CharField(max_length=200)     
+    password=models.CharField(max_length=50)    
```
password는 추후에 암호화해서 저장해야 하기 때문에 넉넉하게 max_length=200으로 수정해주세요~

lordmyshepherd commented on students/juhyeong/users/models.py:
```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
```
데이터의 이력 관리를 위해서 `created_at`, `updated_at` column을 추가해주세요. 
```python 
created_at = models.DateTimeField(auto_now_add=True)
 updated_at = models.DateTimeField(auto_now=True) 
```

lordmyshepherd commented on students/juhyeong/users/models.py:
```diff
@@ -0,0 +1,12 @@
+    password=models.CharField(max_length=50)    
+    phone_number=models.CharField(max_length=11)    
+    profile=models.CharField(max_length=255)
+    class Meta:
```
line 9와 line10 사이에 한 줄 띄워주세요!

BnDC committed:
[MODIFIED] models.py
 - 불필요한 주석 삭제
 - email field   : unique 속성 추가
 - password field: max_length 200으로 수정
 - Alignment convention에 맞게 수정

lordmyshepherd commented:
LGTM 👍
